shell/gcloud-pull-staging-files.sh

```diff
@@ -7,6 +7,9 @@
 ./google-cloud-sdk/bin/gsutil cp "gs://${GCLOUD_BUCKET_DEV}/${GOOGLE_ORG_WHITELIST_JSON_FILE}" ./
 ./google-cloud-sdk/bin/gsutil cp "gs://${GCLOUD_BUCKET_DEV}/${MANAGED_SERVICE_ACCOUNTS_JSON_FILE}" ./
 ./google-cloud-sdk/bin/gsutil cp "gs://${GCLOUD_BUCKET_DEV}/${DEV_DATASET_JSON_FILE}" ./
+echo ${GCLOUD_BUCKET_DEV}
+echo ${DEV_DATASET_JSON_FILE}
+cat ${GCLOUD_BUCKET_DEV}/${DEV_DATASET_JSON_FILE}
```

Comment:
Note, this is running on the CircleCI VM so it would just be `cat ./${DEV_DATASET_JSON_FILE}`

Reply:
Yeah, of course. What was I thinking?